lib/absinthe/federation/schema/phase/add_federated_directives.ex

```diff
@@ -84,6 +90,54 @@ defmodule Absinthe.Federation.Schema.Phase.AddFederatedDirectives do
 
   defp maybe_add_extends_directive(node, _meta), do: node
 
+  defp maybe_add_requiresScopes_directive(node, %{requires_scopes: scopes, absinthe_adapter: adapter}) do
+    directive = Directive.build("requires_scopes", adapter, scopes: scopes)
+
+    add_directive(node, directive)
+  end
+
+  defp maybe_add_requiresScopes_directive(node, _meta), do: node
```

Comment:
```suggestion
  defp maybe_add_requires_scopes_directive(node, _meta), do: node
```

we could name this full snake case

Reply:
fixed